corehq/ex-submodules/pillowtop/processors/form.py

```diff
@@ -34,7 +34,7 @@ def process_change(self, pillow_instance, change):
             # the same effect, an app having has_submissions set to True.
             mark_has_submission(domain, build_id)
 
-        user_id = doc.get('user_id')
+        user_id = doc.get('form', {}).get('meta', {}).get('userID')
```

Comment:
i tested it in sql and it seems to work. `doc` should be the xform itself (which is why later on we can grab meta from it anyway), and the other method definitely doesnt work for couch (not sure if it does for sql)